community_images/grafana/ironbank/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+
+services:
+  prometheus:
+    image: prom/prometheus:v2.17.1
```

Comment:
is it possible to use Rapidfort's prometheus image here?
https://hub.docker.com/r/rapidfort/prometheus

Reply:
Yes, we can. I have replaced the line and all the hardening steps are working fine.